static/js/apps/homepage/components/tools.tsx

```diff
@@ -65,8 +65,8 @@ export const Tools = ({ routes }: ToolsProps): ReactElement => {
             ${theme.typography.text.lg};
           `}
         >
-          Data Commons offers data exploration tools and cloud-based
-          APIs to access and integrate cleaned datasets.
+          Data Commons offers data exploration tools and cloud-based APIs to
```

Comment:
Sorry, this should get cleaned up in https://github.com/datacommonsorg/website/pull/5824